mon-pix/app/components/authentication/oidc-reconciliation.js

```diff
@@ -0,0 +1,47 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+
+export default class OidcReconciliationComponent extends Component {
+  @service oidcIdentityProviders;
+
+  get identityProviderOrganizationName() {
+    return this.oidcIdentityProviders[this.args.identityProviderSlug]?.organizationName;
+  }
+
+  get shouldShowEmail() {
+    return !!this.args.email;
+  }
+
+  get shouldShowUsername() {
+    return !!this.args.username;
+  }
+
+  get shouldShowGarAuthenticationMethod() {
```

Comment:
Suggestion pour la suite : on pourrait essayer de factoriser et d'avoir que `get shoudShowAuthenticationMethod(this.args.authenticationMethods)`
Question déjà posée plus bas : il peut en avoir plusieurs ?

Reply:
vu ensemble : oui c'est prévu qu'on mutualise aussi cette partie de code, c'est pas encore géré